interface.js

```diff
@@ -0,0 +1,157 @@
+var session_key;
+var user_id;
+
+$(document).ready(function(){
+  var response;
+  var i;
```

Comment:
This doesn't need to be defined here, as you are defining it in your for loop with 
`var i = 0`